City-of-Helsinki/drupal-hdbt
# UHF-3511: Add user-edited-content class to wrap custom html

# [UHF-3511](https://helsinkisolutionoffice.atlassian.net/browse/UHF-3511)
Add user-edited-content class to wrap custom html.

## What was done
* Add `user-edited-content` class to templates where custom (editor-written or imported) html is displayed
* Add styles for the class

## How to install

* Make sure your instance is up and running on latest dev branch.
  * `git pull origin dev`
  * `make fresh`
* Update the HDBT theme
  * `composer require drupal/hdbt:dev-UHF-3511-user-edited-content-class`
* Run `make drush-cr`

## How to test

* Add landing page or standard page
  * Add some paragraphs to it you know contain custom HTML fields (long text fields with ckeditor)
  * Inspect the page where the custom HTML is, it should be wrapped with `user-edited-content`
* Read code, can you think of any other places where there is custom HTML?

## Designers review



## Other PRs


* [Rekry](https://github.com/City-of-Helsinki/drupal-helfi-rekry/pull/277)


[UHF-3511]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-3511?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
